data/router.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const Posts = require('./db.js');
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+    try {
+        const posts = await Posts.find(req.query);
+        res.status(200).json(posts);
+    } catch (error) {
+        console.log(error);
+        res.status(500).json({ message: 'Error retreiving the posts.' });
+    };
+});
+
+router.get('/:id', async (req, res) => {
+    try {
+        const post = await Posts.findById(req.param.id);
+        if (post) {
+            res.status(200).json(post);
+        } else {
+            res.status(404).json({ message: 'Post not found.' })
+        };
+    } catch (error) {
+        console.log(error);
+        res.status(500).json({ message: 'Error retreiving the post.' });
+    };
+});
+
+router.post('/', async (req, res) => {
+    try {
```

Comment:
Should see a conditional here to test if they're sending a `req.body.title` and `req.body.content`